src/Poster/Poster.stories.js

```diff
@@ -0,0 +1,230 @@
+      type: "figma",
+      url: "https://www.figma.com/file/Kmd4utmJFPRMVeCFEEBQhLtx/Decanter-Design-System?node-id=16530%3A326",
+    },
+    docs: {
+      description: {
+        component: "More information coming soon...",
+      },
+    },
+  },
+  argTypes: {
+    layout: {
+      type: "select",
+      options: layoutOptions,
+    },
+  },
+};
+
+const PosterTemplate = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Content>
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateImage = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mb-2">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+    <Poster.Content className="su-items-center su-text-center su-align-center">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateImageLeft = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mr-4">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+
+    <Poster.Content className="su-items-start md:su-flex-grow">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
+        Cras varius. Ut non enim eleifend felis pretium feugiat. Suspendisse
+        feugiat. Maecenas egestas arcu quis ligula mattis placerat. Suspendisse
+        faucibus, nunc et pellentesque egestas, lacus ante convallis tellus,
+        vitae iaculis lacus elit id tortor.
+      </p>
+    </Poster.Content>
+  </Poster>
+);
+
+const PosterTemplateCustom = ({ ...rest }) => (
+  <Poster {...rest}>
+    <Poster.Image className="su-rs-mb-2">
+      <img
+        src="https://placekitten.com/600/400"
+        className="su-object-cover su-w-full su-h-full su-object-center"
+        alt=""
+      />
+    </Poster.Image>
+    <Poster.Content className="su-text-white su-items-center su-text-center su-align-center">
+      <Poster.Headline>Title</Poster.Headline>
+      <p className="su-big-paragraph su-leading-snug">
```

Comment:
If you could please add a `su-mb-0` to last paragraphs in all the examples, that'd be great.